sdks/python/run_pylint.sh

```diff
@@ -29,6 +29,12 @@ BASE_BRANCH=python-sdk
 set -e
 set -o pipefail
 
+# This script runs pylint on changed files only and uses git for that purpose.
+# Skip this step if git is not available or this is not a valid git repository.
+(command -v git && git remote) || {
+  echo "Not running pylint. Not in a git repository."; exit 0; 
```

Comment:
there are other failure modes, like the assumption that the remote is named `origin` when in fact the contributors guide recommend `apache`. Not sure exactly what the right answer is here.